dist/services/StructuralMetadataUtils.js

```diff
@@ -156,51 +152,39 @@ function () {
     value: function buildSMUI(allItems, duration) {
       var _this2 = this;
 
-      // Regex to match hh:mm:ss, mm:ss OR seconds(as a float)/minutes(as an int)
-      var regexHHMMSS = /^([0-9]*:[0-9]*:[0-9]*.[0-9]*)$/i;
-      var regexMMSS = /^([0-9]*:[0-9]*)$/i;
-      var regexSS = /^([0-9]*.[0-9]*)$/i; // Convert file duration to seconds
-
+      // Convert file duration to seconds
       var durationInSeconds = Math.round(duration / 10) / 100; // Convert time to HH:mm:ss.ms format to use in validation logic
 
       var convertToSeconds = function convertToSeconds(time) {
-        if (regexMMSS.test(time)) {
-          var _time$split = time.split(':'),
-              _time$split2 = (0, _slicedToArray2["default"])(_time$split, 2),
-              minutes = _time$split2[0],
-              seconds = _time$split2[1];
+        var _ref = [],
+            hours = _ref[0],
+            minutes = _ref[1],
+            seconds = _ref[2]; // Check ss/ss.ms values
+
+        if (Number(time)) {
+          var secondsNum = parseFloat(time);
+          return secondsNum;
+        } else {
+          var _time$split$reverse = time.split(':').reverse();
 
-          var minutesInS = parseInt(minutes) * 60;
-          var secondsNum = seconds ? parseFloat(seconds) : 0.0;
-          return minutesInS + secondsNum;
-        } else if (regexHHMMSS.test(time)) {
-          var _time$split$reverse = time.split(':').reverse(),
-              _time$split$reverse2 = (0, _slicedToArray2["default"])(_time$split$reverse, 3),
-              _seconds = _time$split$reverse2[0],
-              _minutes = _time$split$reverse2[1],
-              hours = _time$split$reverse2[2];
+          var _time$split$reverse2 = (0, _slicedToArray2["default"])(_time$split$reverse, 3);
 
+          seconds = _time$split$reverse2[0];
+          minutes = _time$split$reverse2[1];
+          hours = _time$split$reverse2[2];
           var hoursInS = hours ? parseInt(hours) * 3600 : 0;
+          var minutesInS = parseInt(minutes) * 60;
```

Comment:
In case there are no minutes, I would handle these just like hours: have a default of 0.